subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -137,36 +133,35 @@ class Libero {
                 goals: ['versions:set'],
                 properties: [newVersion: config.nextVersion, generateBackupPoms: "false"])
 
-        // Check if tag exists already
-        List tags = git.tag.list()
-        if (tags.contains(config.releaseName)) {
-            println "Tag: ${config.releaseName} already exists. Either this version has already been released, or it failed to complete before"
-            System.exit(1)
+        if (config.mavenSettings) {
+            releaseVersionRequest.setUserSettingsFile(config.mavenSettings)
+            devVersionRequest.setUserSettingsFile(config.mavenSettings)
         }
+
         mavenResult = maven.execute(releaseVersionRequest)
         if (mavenResult.getExitCode() != 0) {
           throw new IllegalStateException( "Failed to update the pom version from ${config.startVersion} to ${config.releaseVersion}" );
         }
         // Pre-Release property updates
-        config.preProps.each { k, v ->
-            util.executeCommand("sed -i '' \"s|<${k}>[^<>]*</${k}>|<${k}>${v}</${k}>|g\" pom.xml", config.projectDir)
-        }
+        updateProps(config.preProps, config.projectDir)
         git.commit(message: "[libero] prepare release ${config.releaseName}", all: true)
-        git.tag.add(name: config.releaseName)
+        try {
+            git.tag.add(name: config.releaseName)
```

Comment:
Not sure really, since the force flag is really just to bypass the confirmation of the options. I just created #29 to address this in the future though